Services/source/housekeeping/housekeeping_service.c

```diff
@@ -32,6 +32,8 @@
 
 #include <redposix.h> //include for file system 
 
+#include "rtcmk.h" //to get time from RTC
```

Comment:
Why not use the mock rtc for now?

Reply:
Because I'm pretty sure I'd have to set the time before I get the time from it. which is odd in this situation. also that include is just from me trying to get the RTC working. It doesn't, so I'm waiting for Athena to be finished as Josh suggested. Currently the timestamp just gets mocked anyway with predictable values

Reply:
Okay, since there's a predictable mock I approve